managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -1536,6 +1538,11 @@ public synchronized void updateLedgersIdsComplete(Stat stat) {
         }
     }
 
+    synchronized OpAddEntry pollPendingAddEntry() {
+        return pendingAddEntries.poll();
```

Comment:
should we make pendingAddEntries  'private' ?

Reply:
I was thinking of doing that but there is some test code that accesses the field directly.

Reply:
You could mark it with `@VisibleForTesting`